wiki/Tournaments/112/Summer_2019/en.md

```diff
@@ -0,0 +1,580 @@
+---
+tags:
+- 112servis
+- 112ST
+- 112ST 2019
+---
+
+# 112's Summer Tournament 2019
+
+![112ST Logo](img/logo.png)
```

Comment:
it looks like this image has these lines in the transparent area

![image](https://user-images.githubusercontent.com/14264992/82152950-3f85c080-982a-11ea-881b-70201fac6a13.png)

was this intended?

Reply:
I don't think so, will clear those lines and replace the image